### PR TITLE
Added pytest, fixed python3 problems

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -7,7 +7,28 @@ on:
...
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Check black formatting
+      uses: jpetrucciani/black-check@19.10b0
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        flake8 app/ --count --ignore=E231,E402,E501,E722,W503 --max-complexity=18 --show-source --statistics
```
Comment (trowik) [inline .github/workflows/main.yml:29]: the max complexity will be lowered to 10 after refactoring the send_mail and import commands

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -7,7 +7,28 @@ on:
...
+    - name: Test with pytest
+      run: |
+        pip install pytest pytest-django
```
Comment (winged) [inline .github/workflows/main.yml:32]: `pip install -r requirements-dev`. But this should only be run in the container - No need to install it in the "outer" environment, as it won't be used.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -7,7 +7,28 @@ on:
...
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Check black formatting
+      uses: jpetrucciani/black-check@19.10b0
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        flake8 app/ --count --ignore=E231,E402,E501,E722,W503 --max-complexity=18 --show-source --statistics
+    - name: Test with pytest
```
Comment (winged) [inline .github/workflows/main.yml:30]: You could create a second job for testing.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -7,7 +7,28 @@ on:
...
+      run: docker-compose up -d
+    - name: Check container status
+      run: docker-compose ps
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```
Comment (winged) [inline .github/workflows/main.yml:22]: Not needed - you run the tests in the container anyway

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -7,7 +7,28 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
     - uses: actions/checkout@v2
-    - uses: jpetrucciani/black-check@19.10b0
+    - name: Build docker-compose
+      run: docker-compose up -d
```
Comment (winged) [inline .github/workflows/main.yml:13]: I'd add a `docker-compose build rmd` here to be sure it's actually being built.

**Review by winged — CHANGES_REQUESTED** [review]
Good work! However the actions can be simplified, as we're using docker-compose for testing.

File: .github/workflows/main.yml (hunk 2 above)
Comment (trowik) [reply .github/workflows/main.yml]: done

File: .github/workflows/main.yml (hunk 3 above)
Comment (trowik) [reply .github/workflows/main.yml:33]: done

File: .github/workflows/main.yml (hunk 4 above)
Comment (trowik) [reply .github/workflows/main.yml]: done

File: .github/workflows/main.yml (hunk 5 above)
Comment (trowik) [reply .github/workflows/main.yml]: done

**Review by winged — APPROVED** [review]

**Pull request merged**